ui.go

```diff
@@ -275,7 +275,7 @@ func ProjectManage() (string, string, error) {
 
 	fmt.Printf("\n%sChoose a project to use for this application.%s\n\n", TERMCYANB, TERMCLEAR)
 	fmt.Printf("%sNOTE:%s This app will make changes to the project. %s\n", TERMCYANREV, TERMCYAN, TERMCLEAR)
-	fmt.Printf("While those changes are reverseable, it would be better to put it in a fresh new project. \n")
```

Comment:
Shouldn't it be 'reversible'. I had to double check that there wasn't other spelling options for it.